tests/foreman/cli/test_contentview.py

```diff
@@ -931,16 +931,19 @@ def test_positive_add_module_stream_filter_rule(self, module_org, target_sat):
         assert filter_info['rules'][0]['id'] == content_view_filter_rule['rule-id']
 
     @pytest.mark.tier1
-    def test_positive_add_custom_repo_by_name(self, module_org, module_product, module_target_sat):
+    @pytest.mark.parametrize('add_by_name', [True, False], ids=['name', 'id'])
+    def test_positive_add_custom_repo_by(
+        self, module_org, module_product, module_target_sat, add_by_name
+    ):
         """Associate custom content to a content view with name
```

Comment:
Probably want to update the docstring here, since I think reading this test can be a bit confusing now without a small amount of clarification. Really like the parametrization though, a nice way to clean up so much of the cruft in this test file :)

Reply:
Yeah, I will update it.